src/app/dao/tests/FirebaseFirestoreDAOMock.spec.ts

```diff
@@ -7,7 +7,7 @@ import { FirebaseCondition, FirebaseDocument, IFirebaseFirestoreDAO } from '../F
 import { MGPMap } from 'src/app/utils/MGPMap';
 import { ObservableSubject } from 'src/app/utils/tests/ObservableSubject.spec';
 import { Time } from 'src/app/domain/Time';
-import { FieldValue, Unsubscribe, UpdateData } from 'firebase/firestore';
+import { FieldValue, Unsubscribe, UpdateData } from '@angular/fire/firestore';
```

Comment:
est ce que du coup maintenant on est uniforme et up-to-date point de vue imports firebase ?

Reply:
uniforme au sens importer que angular/fire: presque, cf. détail plus bas